docs/UserGuide.md

```diff
@@ -179,6 +246,45 @@ Expected outcome:
 
 ### Deleting a task from a selected internship application : `deltask`
 
+### Adding a tag to an internship application : `addtag`
```

Comment:
Perhaps a screenshot for either addtag or deltag (although not both) would help the user to visualize what would happen if this command is executed?

Reply:
I think I will add it when all the PRs are merged.